js/script.js

```diff
@@ -1,18 +1,101 @@
+// Mobile Menu Variables
 const containerMenu = document.querySelector('.menu');
 const closeMenu = document.querySelector('.menu-close');
 const openMenu = document.querySelector('.menu-open');
 const linkMenu = document.querySelectorAll('.menu a');
 
+// Modal Variables
+const containerModal = document.querySelector('.modal');
+const openModal = document.querySelectorAll('.modal-open');
+
+const infoModalImage = [];
+document.querySelectorAll('.picture > img').forEach((item) => {
+  infoModalImage.push(item.getAttribute('src'));
+});
+
+const infoModalTitle = [];
+document.querySelectorAll('.project-name').forEach((item) => {
+  infoModalTitle.push(item.textContent);
+});
+
+const infoModalLanguage = ['HTML', 'CSS', 'JavaScript', 'Ruby On Rails', 'GitHub'];
+const infoModalDescription = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi <br> <br> Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.';
+const infoModalliveUrl = '#';
+const infoModalseeUrl = '#';
+
+const infoModal = {};
+for (let i = 0; i < 3; i += 1) {
+  infoModal[`infoModal${i}`] = {};
+  infoModal[`infoModal${i}`].image = infoModalImage[i];
+  infoModal[`infoModal${i}`].title = infoModalTitle[i];
+  infoModal[`infoModal${i}`].language = infoModalLanguage;
+  infoModal[`infoModal${i}`].description = infoModalDescription;
+  infoModal[`infoModal${i}`].liveUrl = infoModalliveUrl;
+  infoModal[`infoModal${i}`].seeUrl = infoModalseeUrl;
+}
+
```

Comment:
- [ ] It will be better to implement the `infoModel` as an array that stores objects. This will make adding more projects to your portfolio easier in the future. Kindly check below for more description ⬇️ 

  In your project, the `infoModal` which hosts the content of your modal is an object that stores other objects. Although this implementation works, in the future it might cause a lot of problems when you start adding more projects. The best approach just like described by the previous reviewer is to have the `infoModal` as an array that hosts objects ⬇️ 

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```